### PR TITLE
Show Output button when there are errors on action commands

### PR DESCRIPTION
**Pull request by OvidiuCode**
Stable release checklist:  

* [ ] Updated the version using `python -m dev set-version {version}` 
* [x] Updated `/docs/changelog.md` with the changes for the release

Pre-release checklist:

* [ ] Updated the **Unreleased** section of `/docs/changelog.md` with the new changes.

### CODE REVIEW

File: sema4ai/vscode-client/src/robo/agentPackage.ts [hunk 1]
```diff
@@ -253,9 +253,8 @@ export const updateAgentVersion = async (agentPath: string): Promise<void> => {
         const errorMsg = `Failed to update the agent at: ${agentPath}`;
         logError(errorMsg, error, "ERR_UPDATE_AGENT_VERSION");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```
Comment (fabioz) [inline sema4ai/vscode-client/src/robo/agentPackage.ts:256]: If `logError(errorMsg)` is used, the `OUTPUT_CHANNEL.appendLine(errorMsg);` is not needed (as `logError` will already add that to the OUTPUT_CHANNEL).

File: sema4ai/vscode-client/src/robo/agentPackage.ts [hunk 2]
```diff
@@ -154,9 +155,8 @@ export const packAgentPackage = async (targetDir: string): Promise<{ zipPath: st
         const errorMsg = `Failed to package the agent at: ${targetDir}`;
         logError(errorMsg, error, "ERR_PACK_ACTION_PACKAGE");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```
Comment (fabioz) [inline sema4ai/vscode-client/src/robo/agentPackage.ts:158]: If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).

File: sema4ai/vscode-client/src/robo/agentPackage.ts [hunk 3]
```diff
@@ -287,8 +286,7 @@ export const refreshAgentSpec = async (agentPath: string): Promise<void> => {
         const errorMsg = `Failed to refresh the Agent Configuration`;
         logError(errorMsg, error, "ERR_REFRESH_AGENT_SPEC");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```
Comment (fabioz) [inline sema4ai/vscode-client/src/robo/agentPackage.ts:289]: If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).

File: sema4ai/vscode-client/src/robo/agentPackage.ts [hunk 4]
```diff
@@ -134,7 +135,7 @@ export const createAgentPackage = async (): Promise<void> => {
         const errorMsg = "Error creating Agent Package at: " + targetDir;
         logError(errorMsg, err, "ERR_CREATE_ACTION_PACKAGE");
         OUTPUT_CHANNEL.appendLine(errorMsg);
```
Comment (fabioz) [inline sema4ai/vscode-client/src/robo/agentPackage.ts:137]: If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).

File: sema4ai/vscode-client/src/robo/actionPackage.ts [hunk 5]
```diff
@@ -1066,7 +1066,7 @@ const getActionPackageStatus = async (
         }
     );
     if (!result.success) {
-        window.showErrorMessage(`Failed to get action package status: ${result.message}`);
+        showErrorMessageWithShowOutputButton("Failed to get action package status");
```
Comment (fabioz) [inline sema4ai/vscode-client/src/robo/actionPackage.ts:1069]: Question (just double-checking): Have you checked that the `result.message` is actually added to the OUTPUT_CHANNEL (in all the scenarios where it was removed)?

**Review by fabioz — CHANGES_REQUESTED** [review]

File: sema4ai/vscode-client/src/robo/actionPackage.ts (hunk 5 above)
Comment (OvidiuCode) [reply sema4ai/vscode-client/src/robo/actionPackage.ts:1069]: yes, on all

**Review by fabioz — APPROVED** [review]

**Pull request merged**